### ISSUE TITLE
Relative paths not working as expected with `parquet_scan` and `csv_scan` in slt tests

### ISSUE THREAD

**Issue by scsmithr** [GlareDB/glaredb#1207]
## Context

```
2023-06-28T00:23:15.631409Z ERROR                 main ThreadId(01) testing::slt::cli: crates/testing/src/slt/cli.rs:249: Error while running test `sqllogictests/functions/parquet_scan` error=test fail: query failed: db error: ERROR: Error during planning: Failed to canonicalize path "./testdata/parquet/userdata1.parquet": No such file or directory (os error 2)
[SQL] select count(*) from parquet_scan('./testdata/parquet/userdata1.parquet')
at /Users/sean/Code/github.com/glaredb/glaredb/testdata/sqllogictests/functions/parquet_scan.slt:10

Error: Test failures
error: test failed, to rerun pass `-p testing --test sqllogictests`
```

**Comment by universalmind303** [comment]
For some additional context, this appears to work fine using the TUI. 
<img width="703" alt="image" src="https://github.com/GlareDB/glaredb/assets/21327470/265a507f-126e-4005-8f79-6967f5c8e7e4">

**Comment by scsmithr** [comment]
I'll need to double check the working directory for the slt stuff.

**Comment by greyscaled** [comment]
Might be addressed by #1242

**Comment by scsmithr** [comment]
Figure out that the working directory was two levels in. SLTs were adapted so no longer an issue.
